apps/frontend/web/src/pages/Forge.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button, Center } from '@chakra-ui/react';
+
+const Forge = () => {
+  const addData = async () => {
+    await fetch('/api/forge', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+    }).then(() => {
```

Comment:
@YDrogen veut pas qu'on fasse des `.then` combinés avec des `async / await`

Reply:
Est-ce que cela est mieux ?

Reply:
const addData = async () => {
    try {
      const res = await fetch('/api/forge', {
        method: 'POST',
        headers: { 'Content-Type': 'application/json' },
      });
    } catch(error) {
      console.error("The promise is rejected !", error);
    }
  };

Reply:
``const addData = async () => {
    try {
      const res = await fetch('/api/forge', {
        method: 'POST',
        headers: { 'Content-Type': 'application/json' },
      });
    } catch(error) {
      console.error("The promise is rejected !", error);
    }
  };``